hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/merkle/MerkleSchemaRegistry.java

```diff
@@ -186,21 +187,18 @@ public void migrate(
                     map.setLabel(StateUtils.computeLabel(serviceName, stateKey));
                     hederaState.putServiceStateIfAbsent(md, map);
                 } else {
-                    final var ks = new OnDiskKeySerializer(md);
-                    final var ds = new JasperDbBuilder()
-                            .maxNumOfKeys(def.maxKeysHint())
-                            .keySerializer(ks)
-                            .virtualLeafRecordSerializer(new VirtualLeafRecordSerializer(
-                                    (short) 1,
-                                    DataFileCommon.VARIABLE_DATA_SIZE,
-                                    ks,
-                                    (short) 1,
-                                    DataFileCommon.VARIABLE_DATA_SIZE,
-                                    new OnDiskValueSerializer(md),
-                                    false));
-
+                    final MerkleDbTableConfig<OnDiskKey, OnDiskValue> tableConfig = new MerkleDbTableConfig<>(
```

Comment:
```suggestion
                    final var tableConfig = new MerkleDbTableConfig<OnDiskKey, OnDiskValue>(
```

Reply:
I don't think it would look cleaner, but I agree it's more consistent with the rest of the file. Let me change it